example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -38,8 +41,15 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import io.honeycomb.opentelemetry.android.example.ui.theme.HoneycombOpenTelemetryAndroidTheme
 import io.opentelemetry.android.OpenTelemetryRum
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.setValue
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.Headers
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import java.io.IOException
+
+val TAG = "example-app"
```

Comment:
The TAG is usually the name of the class, by convention. I would use `MainActivity` here.